src/test/features/response/routes/view-reponse.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import { checkAuthorizationGuards } from './checks/authorization-check'
+
+import { Paths as ResponsePaths } from 'response/paths'
+
+import { app } from '../../../../main/app'
+
+import * as idamServiceMock from '../../../http-mocks/idam'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
+
+import { checkCountyCourtJudgmentRequestedGuard } from './checks/ccj-requested-check'
+import { checkNotDefendantInCaseGuard } from './checks/not-defendant-in-case-check'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const pagePath: string = ResponsePaths.viewResponsePage.evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })
+
+describe('Defendant response: confirmation page', () => {
```

Comment:
We should rename it, 'confirmation page' makes me think that this is for the confirmation page after successful action was made e.g. response submitted.